src/output-json-stats.c

```diff
@@ -227,8 +228,17 @@ json_t *StatsToJSON(const StatsTable *st, uint8_t flags)
     uint32_t u = 0;
     if (flags & JSON_STATS_TOTALS) {
         for (u = 0; u < st->nstats; u++) {
+            bool is_exception_policy_counter = false;
             if (st->stats[u].name == NULL)
                 continue;
+            if (strstr(st->stats[u].name, "exception_policy") != NULL) {
```

Comment:
wonder if this may be a big high overhead. Can we have a stat flag like "skip on 0" or something?

Reply:
For all stats, or just for Exception policies?

Reply:
If for all stats, then the question that arises is: aren't there stats that are important to be logged, even if counters are zero?

I _think_ that it's not difficult to not log all stats counters that are zero.

Reply:
Went with that, for now.